### PR TITLE
Add HMAC and KID fields to ACME config TOML for EAB integration.

### PR DESCRIPTION
**Pull request by banaag**
Add HMAC and KID fields to ACME config TOML for EAB integration.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -27,6 +27,8 @@ replace github.com/davecgh/go-spew => github.com/davecgh/go-spew v1.1.0
 
 replace github.com/stretchr/testify => github.com/stretchr/testify v1.2.1
 
+replace github.com/grpc/grcp-go => github.com/grpc/grpc-go v1.33.1
```
Comment (twifkak) [inline go.mod:30]: Did you add this manually? If so, why (OOC)?

File: packager/util/config.go [hunk 2]
```diff
@@ -66,10 +66,17 @@ type ACMEConfig struct {
...
+	AccountURL string
+	// ACME Account URL. If non-empty, we will auto-renew cert via ACME.
+	DiscoURL string
+	// Email address registered with ACME CA.
+	EmailAddress string
+	// Key Identifier from ACME CA. Used for External Account Binding.
+	EABKid string
+	// MAC Key from ACME CA. Used for External Account Binding. Should be in
+	// Base64 URL Encoding without padding format.
+	EABHmac string
```
Comment (twifkak) [inline packager/util/config.go:79]: Add documentation in amppkg.example.toml? (Or make sure it's on your TODO list.)

**Review by twifkak — APPROVED** [review]

File: packager/util/config.go (hunk 2 above)
Comment (banaag) [reply packager/util/config.go:79]: yeah this will be in the next PR.

File: go.mod (hunk 1 above)
Comment (banaag) [reply go.mod:30]: Oh good catch.  This was failing for me before you did your check-in for the release.  I think it's probably fine to remove it now, will do it if it checks out.

**Review by twifkak — APPROVED** [review]

File: packager/util/config.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (banaag) [reply packager/util/config.go:79]: Added the check in the certfetcher.

**Review by twifkak — APPROVED** [review]

**Pull request merged**